src/test/java/com/solvd/movie/service/impl/EsMovieServiceTest.java

```diff
@@ -16,18 +16,16 @@
 @ExtendWith(MockitoExtension.class)
 public class EsMovieServiceTest {
 
-    private final FakeMovieService fakeMovieService = new FakeMovieService();
-
     @Mock
-    private EsRepository esRepository;
+    private EsMovieRepository esMovieRepository;
 
     @InjectMocks
     private EsMovieServiceImpl esMovieService;
 
     @Test
     public void retrieveAll() {
-        EsMovie movie = this.fakeMovieService.getEsMovie();
-        Mockito.when(this.esRepository.findAll())
+        EsMovie movie = FakeMovieService.getEsMovie();
```

Comment:
@hizmailovich I can't see any sense in the usage of fake objects here, only repository I think can be `faked`

fakes are used instead of mocking dependencies directly, this is the idea